<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stadium Manager</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* === Main Window &amp; Sidebar === */
#centralwidget {
    background-color: #3e4a59; /* A slightly lighter dark for the main area */
}
#verticalLayoutWidget {
    background-color: #2c3e50; /* Dark background for the sidebar */
}

/* === Make the container for pages transparent (CRUCIAL FIX) === */
#centralArea {
    background-color: transparent;
}

/* === Sidebar Buttons === */
#verticalLayoutWidget QPushButton {
    background-color: transparent;
    color: #e0e0e0;
    padding: 10px;
    text-align: left;
    font-size: 14px;
    border-radius: 5px;
    border: none;
    font-family: Segoe UI, Arial, sans-serif;
}
#verticalLayoutWidget QPushButton:hover {
    background-color: #34495e;
}
#verticalLayoutWidget QPushButton:checked {
    background-color: #0d6efd;
    color: #ffffff;
    font-weight: bold;
}

/* === Sidebar Sub-menu buttons === */
#widgetManage QPushButton {
    padding-left: 25px;
    font-size: 13px;
    color: #bdc3c7;
}

/* === Logout Button === */
#btnLogOut {
    color: #e74c3c;
    font-weight: bold;
}
#btnLogOut:hover {
    background-color: #e74c3c;
    color: #ffffff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="MenuBar">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnHomePage">
       <property name="text">
        <string>Home</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnManageSport">
       <property name="text">
        <string>Sân</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBookedFields">
       <property name="text">
        <string>Các sân đang đặt</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnManage">
       <property name="text">
        <string>Quản lý</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetManage" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnQLKH">
          <property name="text">
           <string>Quản lý khách hàng</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnQLNS">
          <property name="text">
           <string>Quản lý nhân sự</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBCTK">
       <property name="text">
        <string>Báo cáo thống kê</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCTK">
       <property name="text">
        <string>Cấp tài khoản</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogOut">
       <property name="text">
        <string>Đăng xuất</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="centralArea" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>1031</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1253</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
